packages/components/src/PhaseCountdown/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { CountdownTimerProps, InjectedCountdownTimerProps, CountdownTimerState } from "./types";
+import { TextCountdownTimerComponent } from "./TextCountdownTimer";
+import { ProgressBarCountdownTimerComponent } from "./ProgressBarCountdownTimer";
+import * as React from "react";
+
+const connectCountdownTimer = () => <TCountdownTimerProps extends CountdownTimerProps>(
+  CountdownTimerComponent:
+    | React.ComponentClass<TCountdownTimerProps & InjectedCountdownTimerProps>
+    | React.StatelessComponent<TCountdownTimerProps & InjectedCountdownTimerProps>,
+) => {
+  return class HOCountdownTimerContainer extends React.Component<TCountdownTimerProps, CountdownTimerState> {
+    public _timer: number | undefined;
```

Comment:
any reason why this is prefixed with an underscore? also it could probably be `public timer?: number` right?